### ISSUE TITLE
linter: regression in oxlint 1.60.0 for jest/no-standalone-expect

### ISSUE THREAD

**Issue by noahnu** [oxc-project/oxc#21844]
### What version of Oxlint are you using?

1.61.0

### What command did you run?

oxlint

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```ts
export default defineConfig({
  options: { typeAware: true },
  plugins: ['eslint', 'vitest'],
  categories: { correctness: 'error' },
  rules: {},
})
```

### What happened?

I re-export a vitest text fixture like so:

`#testUtils`:

```
import { test as baseTest } from 'vitest'

export const it = baseTest.extend({
  fixture: async (_, use) => {
    await use()
  }
})
```

and then in my test file:

```
import { it } from '#testUtils'

describe('Test', () => {
  it('passes', ({ fixture }) => {
    expect(true).toBe(true)
  })
})
```

In oxlint 1.60.0 this worked. In oxlint 1.61.0 I receive this error:

```
`expect` must be inside of a test block.
help: Did you forget to wrap `expect` in a `test` or `it` block? oxc[eslint-plugin-jest(no-standalone-expect)](https://oxc.rs/docs/guide/usage/linter/rules/jest/no-standalone-expect.html)
```

**Comment by camc314** [comment]
Thanks for reporting, I checked this against the upstream eslint rule, and it reports the same patter by default whtn `it` is imported from a custom helper module rather than from `vitest`.

For this setup, configure the rule with `additionalTestBlockFunctions`, for example:

```json
{
  "rules": {
    "vitest/no-standalone-expect": ["error", {
      "additionalTestBlockFunctions": ["it"]
    }]
  }
}
